spring-boot/admin/src/main/resources/templates/admin/fragments/wizard/wizard.html

```diff
@@ -1,348 +1,381 @@
+                                            <input type="hidden" id="percallChargeInput" name="percallCharge"/>
+
+                                            <div id="amountDiv" style="display: none;">
+                                                <h2 class="StepTitle"></h2>
+                                                <p>
+                                                <h2> How much should your service users pay (&euro; per month or per
+                                                    1000 calls)?</h2>
+                                                </p>
+                                                <input id="serviceCost" type="number" name="serviceCost" min="0"
+                                                       step="0.01" class="form-control col-md-12 col-xs-12"/>
                                             </div>
                                         </div>
+                                        <br/><br/>
+                                        <h2 id="clickNextButtonMessage">Please click the "Next" button below to
+                                            continue.</h2>
                                     </div>
-                                </form>
+                                    <div class="col-md-5 col-xs-12">
+                                        <img src="/images/getting-started-providers/coatrack-wizard-service-pricing.png"
+                                             style="width:100%;max-width:390px;"/>
+                                    </div>
+                                </div>
+                                <div id="step-5">
+                                    <div class="col-md-7 col-xs-12">
+                                        <h1>Summary </h1>
+                                        <div class="divider"></div>
+                                        Service name:
+                                        <h2 id="serviceNameSummary"/>
+                                        (Local) Service URL:
+                                        <h2 id="serviceUrlSummary"></h2>
+                                        Payment/Pricing:
+                                        <h2 id="costMethod"></h2>
+                                        <div class="clearfix"></div>
+                                        <div class="divider"></div>
+                                        <br/>
+                                        &nbsp;
+                                        <br/>
+                                        <h2>Click the Finish button to generate your custom CoatRack Gateway.</h2>
+                                    </div>
+                                    <div class="col-md-5 col-xs-12">
+                                        <img src="/images/getting-started-providers/coatrack-wizard-overview.png"
+                                             style="width:100%;max-width:390px;"/>
```

Comment:
I did not yet check all the HTML changes up to this line in detail, my assumption is they are just adding newlines, but not changing code?!

Reply:
This was a typo.
Done

Reply:
Sorry if my question was not clear: did you change any HTML up to line 200? It looks like a lot of changes, but I assume it was just auto-indentation?